backend/test/integrated/group.js

```diff
@@ -42,6 +42,7 @@ describe("Groups", function () {
 			{
 				"account":"groupUser",
 				"model":"4ec71fdd-0450-4b6f-8478-c46633bb66e3",
+				"ifc_guids":[],
```

Comment:
It should really be one or the other no? either returns ifc_guids, or shared ids.